dashboard/forms.py

```diff
@@ -274,6 +274,23 @@ class Meta:
         fields = ['what']
 
 
+class MABEditCandidateForm(forms.Form):
+    candidate = forms.ModelChoiceField(queryset=Brother.objects.filter(brother_status='0'), required=True)
+
+
+class MeetABrotherEditForm(forms.Form):
+    update = forms.BooleanField(label="", required=False)
+
+    def __init__(self, *args, **kwargs):
+        brother = Brother.objects.get(pk=kwargs.pop('brother', ""))
+        exists = kwargs.pop('mab_exists', "")
+        super(MeetABrotherEditForm, self).__init__(*args, **kwargs)
+
+        if brother:
```

Comment:
Should probably explain what is happening here

```suggestion
        # add the label and exists fields of the brother if it exists
        if brother:
```